src/AssetBundle.php

```diff
@@ -105,22 +170,66 @@ class AssetBundle
      *   pairs that will be used to overwrite {@see $jsOptions} settings for this entry.
      *
      * Note that only a forward slash "/" should be used as directory separator.
+     *
+     * @psalm-var JsFile[]
      */
     public array $js = [];
 
+    /**
+     * @var array List of JS blocks. Each JS block can be specified in one of the following formats:
+     *
+     *  - JS block. For example, `alert(42);`.
+     *  - An array, with the first entry is JS block, second entry is position on page (integer, optional) that
+     *    will be used to overwrite {@see $jsPosition} setting and a list of key/value pairs
+     *    that will be used to overwrite {@see $jsOptions} settings for this CSS block.
+     *
+     * Optionally, use string keys for identifies the JS block.
+     *
+     * Example:
+     *
+     * ```php
+     * public array $jsStrings = [
+     *     'alert(1);',
+     *     ['alert(2);'],
+     *     ['alert(3);', 3],
+     *     ['alert(4);', 3, 'id' => 'main'],
+     *     'key1' => 'alert(5);',
+     *     'key2' => ['alert(6);'],
+     *     'key3' => ['alert(7);', 3],
+     *     'key4' => ['alert(8);', 3, 'id' => 'second'],
+     * ];
+     * ```
+     *
+     * @psalm-var JsString[]
+     */
+    public array $jsStrings = [];
+
     /**
      * @var array The options that will be passed to {@see \Yiisoft\View\WebView::registerJsFile()}
      * when registering the JS files in this bundle.
      */
     public array $jsOptions = [];
 
     /**
-     * @var array JavaScript code blocks to be passed to {@see \Yiisoft\View\WebView::registerJs()}.
+     * @var int|null JavaScript files and blocks position.
```

Comment:
Probably need to specify in docblock what `int ' stands for, what possible values are.

Reply:
Done.